recipes/snakemake/meta.yaml

```diff
@@ -30,7 +30,7 @@ requirements:
     - docutils
     - pyyaml
     - requests >=2.8.1
-    - dropbox >=5.2
```

Comment:
I'm not even sure this is needed, just wanted to point this out, that you are not surprised to not see a new package. Maybe you can add a comment to those changes so that the maintainer knows about this later on.